webapp/src/containers/Rhs/projectDetails/index.tsx

```diff
@@ -111,9 +122,9 @@ const ProjectDetails = (projectDetails: ProjectDetails) => {
                 isOpen={showSubscriptionConfirmationModal}
                 onHide={() => setShowSubscriptionConfirmationModal(false)}
                 onConfirm={handleConfirmDeleteSubscription}
-                isLoading={usePlugin.getApiState(plugin_constants.pluginApiServiceConfigs.deleteSubscription.apiServiceName, subscriptionToBeDeleted).isLoading}
+                isLoading={getApiState(plugin_constants.pluginApiServiceConfigs.deleteSubscription.apiServiceName, subscriptionToBeDeleted).isLoading}
                 confirmBtnText='Delete'
-                description='Are you sure you want to delete this subscription?'
+                description='Are you sure you want to delete this subscription ?'
```

Comment:
Why this change?

Reply:
this may have occurred due to syncing branches